commons-hpcc/pom.xml

```diff
@@ -157,6 +157,20 @@
             </build>
         </profile>
     </profiles>
+    <developers>
```

Comment:
Let's either leave this off, or include James and Michael

Reply:
Removed